pages/profile.js

```diff
@@ -3,187 +3,164 @@ import Link from 'next/link'
+  const [viewOASFields, setViewOASFields] = useState(false)
+
+  const benefitInformations = [
+    {
+      show: viewCPPFields,
+      setShow: setViewCPPFields,
+      fields: fakeFieldsCPP,
+      benefit: t.cpp,
+    },
+    {
+      show: viewEIFields,
+      setShow: setViewEIFields,
+      fields: [...fakeFieldsEI, ...fakeFieldsEI2],
+      benefit: t.ei,
+    },
+    {
+      show: viewOASFields,
+      setShow: setViewOASFields,
+      fields: fakeFieldsOAS,
+      benefit: t.oas,
+    },
+  ]
   return (
     <>
       <h1 className="py-4 text-4xl font-bold text-gray-darker">
-        {t.profileAndSecuritySettings}
+        {t.profileSettings}
       </h1>
-      <TabList
-        titles={[t.profileAndPreferences, t.securitySettings]}
-        onClick={(index) => setTabSelected(index)}
-        tabSelected={tabSelected}
-        containerStyle={'flex flex-wrap border-b border-gray-200'}
-        selectedTabStyle={
-          'bg-gray-100 active border-b-2 border-b-bright-blue-light'
-        }
-        unselectedTabStyle={
-          'text-gray-500 hover:text-gray-600 hover:bg-gray-50'
-        }
-        genericTabStyle={'inline-block text-lg py-3 px-4 text-center'}
-        locale={props.locale}
-      />
-      {tabSelected === 0 ? (
-        <>
-          <TabList
-            titles={[t.ei, t.cpp, t.oas]}
-            onClick={(index) => setProfileTabSelected(index)}
-            tabSelected={profileTabSelected}
-            containerStyle={'my-4 flex flex-wrap gap-4'}
-            selectedTabStyle={'bg-bright-blue-lighter'}
-            unselectedTabStyle={'bg-gray-lighter'}
-            genericTabStyle={
-              'text-center px-10 py-1 whitespace-nowrap rounded-xl hover:bg-bright-blue-lighter '
-            }
-            locale={props.locale}
-          ></TabList>
-          {profileTabSelected === 0 || !profileTabSelected ? (
-            <ProfileInfo
-              fields={[fakeFieldsEI, fakeFieldsEI2]}
-              locale={props.locale}
-            />
-          ) : profileTabSelected === 1 ? (
-            <ProfileInfo fields={[fakeFieldsCPP]} locale={props.locale} />
-          ) : profileTabSelected === 2 ? (
-            <ProfileInfo fields={[fakeFieldsOAS]} locale={props.locale} />
-          ) : null}
-        </>
-      ) : tabSelected === 1 ? (
-        <ProfileInfo fields={[fakeSecurityFields]} locale={props.locale} />
-      ) : null}
+      <hr className="border-1 border-red-400" />
+      <span className="text-lg">{t.updateProfile}</span>
+
+      {benefitInformations.map((benefitInfo) => {
+        return (
+          <>
+            <h2 className="text-3xl py-2 font-bold">{benefitInfo.benefit}</h2>
+            <div className="py-2 mb-4">
+              {benefitInfo.show ? (
+                <ProfileInfo
+                  fields={benefitInfo.fields}
+                  locale={props.locale}
+                />
+              ) : null}
+              <ViewMoreLessButton
```

Comment:
Double check the styling on this, it may need an option to override some styles.

Reply:
Ah yeah, wasn't sure if I should've added something like that. Done, removed the left margin of the buttons on the page

Reply:
The padding and margins in theses sections still seem quite off from the designs.

Reply:
Seems like the `pl-0` class doesn't apply when it comes after `pl-2`, so I removed the padding out of the component.